stack-orchestration/configuration/di-id-reuse-core-build-admin/ais-main-pipeline/parameters.json

```diff
@@ -29,7 +29,7 @@
   },
   {
     "ParameterKey": "CustomKmsKeyArns",
-    "ParameterValue": "arn:aws:kms:eu-west-2:216552277552:key/4bc58ab5-c9bb-4702-a2c3-5d339604a8fe,arn:aws:kms:eu-west-2:688341086169:key/d357b3a0-6129-46bc-9b64-6ab9ae155007,arn:aws:kms:eu-west-2:688341086169:key/ced952f4-921e-4233-b209-7481ca55e532"
+    "ParameterValue": "arn:aws:kms:eu-west-2:216552277552:key/4bc58ab5-c9bb-4702-a2c3-5d339604a8fe,arn:aws:kms:eu-west-2:688341086169:key/d357b3a0-6129-46bc-9b64-6ab9ae155007,arn:aws:kms:eu-west-2:688341086169:key/ced952f4-921e-4233-b209-7481ca55e532,arn:aws:kms:eu-west-2:688341086169:key/ced952f4-921e-4233-b209-7481ca55e532"
```

Comment:
Hey! Would you be able to please also remove the last ARN - `arn:aws:kms:eu-west-2:688341086169:key/ced952f4-921e-4233-b209-7481ca55e532` as its a duplicate, I've also removed it from the pipeline parameters in build now too - thank you 🙏

Reply:
removed